cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java

```diff
@@ -948,6 +953,7 @@ TSStatus processPlanLocally(PhysicalPlan plan) {
 
       log.setPlan(plan);
       plan.setIndex(log.getCurrLogIndex());
+      log.setByteSize(log.serialize().array().length);
```

Comment:
According to my previous test results, the `serialize()` step doesn't take very little overhead. Since we need serialization to calculate the memory, the leader should try to reuse the serialization here with the one sent to the followers to reduce overhead. What's your opinion?

Reply:
ok, I will try reuse it.